examples/guided_onboarding_example/public/components/main.tsx

```diff
@@ -199,7 +206,7 @@ export const Main = (props: MainProps) => {
         </EuiText>
         <EuiSpacer />
         <EuiFlexGroup>
-          {(Object.keys(guidesConfig) as GuideId[]).map((guideId) => {
+          {(['search', 'security', 'observability', 'testGuide'] as GuideId[]).map((guideId) => {
```

Comment:
[Outside scope of this PR] This made me think of the cards on the landing page. We may have to rethink how that is rendered 🤔 . Right now, I believe the "cards" content is hard-coded. Are we going to expect each solution to update that component if a new guide is added?

Reply:
You're right, we need to consider the landing page cards, maybe allow plugins register a guide card and/or a link card. I created an issue for that [here](https://github.com/elastic/kibana/issues/146672).